### PR TITLE
[develop] -i support for relative paths

### PR DESCRIPTION
**Pull request by NorseGaud**
## Change Description

https://github.com/EOSIO/eos/issues/7560

Base Images (with BATS)
https://buildkite.com/EOSIO/eosio-base-images/builds/368

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: scripts/eosio_uninstall.sh [hunk 1]
```diff
@@ -1,89 +1,125 @@
...
+VERSION=2.1
+# User input handling
+DEP_PROCEED=false
+FULL=false
+
+function usage() {
+   printf "Usage: $0 OPTION...
+  -i DIR      Directory where eosio is installed)
+  -y          Noninteractive mode (answers yes to every prompt)
+  -f          Removal of data directory (be absolutely sure you want to delete it before using this!)
```
Comment (arhag) [inline scripts/eosio_uninstall.sh]: Please update the README as well to reflect these changes to the uninstall script.

File: scripts/eosio_uninstall.sh [hunk 2]
```diff
@@ -1,89 +1,125 @@
...
+      read -p "By specifying -f, removal of the eosio data directory will require a resync of data which can take days. Do you wish to proceed? (y/n) " PROCEED
+      case $PROCEED in
+         "" ) echo "What would you like to do?";;
+         0 | true | [Yy]* ) break;;
+         1 | false | [Nn]* ) exit 0;;
+         * ) echo "Please type 'y' for yes or 'n' for no.";;
+      esac
+   done
+elif $NONINTERACTIVE && $FULL; then
+   echo "${COLOR_RED}By specifying -f, removal of the eosio data directory will require a resync of data which can take days. The script is sleeping for 15 seconds to give you a chance to cancel if this is not what you want.${COLOR_NC}"
```
Comment (arhag) [inline scripts/eosio_uninstall.sh]: If the user has specified `-y` for non-interactive (which is not the default), I don't think we should be waiting in the script for a user action that is likely not going to come.

File: README.md [hunk 3]
```diff
@@ -68,9 +68,10 @@ $ sudo yum remove eosio
 
 #### Build Script Uninstall
 
-If you have previously installed EOSIO using build scripts, you may execute `./scripts/eosio_uninstall.sh` to uninstall.
-- Passing `--force` will answer yes to all prompts
-- Passing `--full` will remove data directories (be very careful with this)
+If you have previously installed EOSIO using build scripts, you can execute `eosio_uninstall.sh` to uninstall.
+- Passing `-y` will answer yes to all prompts (forces `-f`)
```
Comment (arhag) [inline README.md]: I don't think `-y` should force `-f`. They should be independent.

`-y` alone should not remove the data directories and not prompt the user for anything. `-f` alone should remove the data directories after prompting the user if they are sure. `-f -y` should proceed forward with removing the data directories without prompting the user.

Comment (NorseGaud) [reply README.md]: Ah, I misunderstood. I'll get that fixed

**Pull request merged**